node/network/availability-distribution/src/lib.rs

```diff
@@ -451,10 +445,11 @@ async fn send_tracked_gossip_messages_to_peers<Context>(
 where
 	Context: SubsystemContext<Message = AvailabilityDistributionMessage>,
 {
-	if peers.is_empty() {
```

Comment:
Is the goal just to modify the internal state in a way that the test can detect? In that case, it might be better to write it like 

```rust
if peers.is_empty() && !cfg!(test) {
```